packages/driver/cypress/fixtures/auth/idp.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+<head>
+</head>
+<body>
+    <label for="name">Username: </label>
+    <input data-cy="username" type="text" name="name" id="name" required>
+    <button data-cy="login">Login</button>
```

Comment:
Tabbing is off here.
```suggestion
  <label for="name">Username: </label>
  <input data-cy="username" type="text" name="name" id="name" required>
  <button data-cy="login">Login</button>
```

Reply:
updated